packages/manager/modules/octavia-load-balancer/src/octavia-load-balancer.html

```diff
@@ -1,2 +1 @@
-<div data-ovh-alert="octavia.alerts.global"></div>
```

Comment:
This should be kept as it is. If it is not available at listing page, create a new one for listing page and use it.

Reply:
Sorry I don't understand, there is one in the listing page. Mockups were showing the alerts under just above the listing so I had to move it in the listing.